docs/review_process.rst

```diff
@@ -0,0 +1,163 @@
+"Checks" section of the PR view.
+
+ghostflow
+---------
+Runs basic checks on the commits submitted in a PR.
+Should ghostflow find any issues with the build, the diagnostics are written
+out prompting the submitter to correct the reported issues.
+If there are no issues, or just warnings, ghostflow simply reports a successful
+build.
+The branch should usually pass this check before it can be merged.
+In the rare case that a PR is not subject to a change note, then failure of
+this check specifically in regard to that lower-level check, may be ignore by
+the reviewer.
+
+Some reports such as whitespace issues will need to be corrected by rewriting
+the commit.
+This is generally handled by performing ``git commit --fixup=...`` commits and
+performing a ``git rebase -i --autosquash ...`` rebase afterwards, or a more
+simple global squash if appropriate.
+
+LGTM Analysis
+-------------
+Runs a more advanced code analysis tool over the branch that can address issues
+that the submitter might not have noticed.
+Should LGTM find an issue, it will write a comment on the PR and provide a link
+to the LGTM website for more detailed information.
+The comment should be addressed by the submitter before continuing to submit
+for review.
+Ideally a submitted PR adds no new issues as reported by LGTM.
+
+Passage of this check is not strictly required but highly encouraged.
+
+SonarCloud Code Analysis
+------------------------
+Similar to LGTM, this service performs a more in-depth analysis of the code.
+This should provide the same output as a SonarQube scan.
+
+Passage of this check is not strictly required but highly encouraged.
+
+lint
+----
+Runs ``flake8`` to quality check the code style.
+You can run this check manually in your local repository with
+``poetry run flake8``.
+
+Passage of this check is strictly required.
+
+MyPy
+----
+Performs static type analysis.
+You can run this check manually in your local repository with ``poetry run
+mypy``.
+
+Passage of this check is strictly required.
+
+Unittests
+---------
+Runs the unittests created under ``tests/`` as well as any doc-tests found in
+doc-strings in the package code proper.
+You can run this check manually  in your local repository with ``poetry run
+pytest``.
+
+Passage of these checks is strictly required.
+
+CodeCov
+-------
+This check reports aggregate code coverage as reported from output of the
+unittest jobs.
+This check requires that all test code be "covered" (i.e. there is no dead-code
+in the tests) and that a minimum coverage bar is met for package code changed
+or added in the PR.
+The configuration for this may be found in the ``codecov.yml`` file in the
+repository root.
+
+Passage of these checks is strictly required.
+
+ReadTheDocs Documentation Build
+-------------------------------
+This check ensures that the documentation portion of the package is buildable
+by the current host ReadTheDocs.org.
```

Comment:
What happens if the ReadTheDocs build fails, i.e. is this check strictly required (keeping in line with the other tests)?

Reply:
Good catch, I've added a similar line as the other "required" checks.